src/Infrastructure/EudonetParis/EudonetParisExecutor.php

```diff
@@ -41,6 +41,8 @@ public function execute(\DateTimeInterface $laterThanUTC): void
         $numSkippedNoLocationsGathered = 0;
         $numErrors = 0;
         $startTime = $this->dateUtils->getMicroTime();
+        $numberOfRegulationsInsideEudonet = 0;
+        $numberOfMeasuresInsideEudonet = 0;
```

Comment:
Initialiser à `null` plutôt ? En cas d'erreur voir "nombre : 0" dans le rapport final peut être confusant

```suggestion
        $numberOfRegulationsInsideEudonet = null;
        $numberOfMeasuresInsideEudonet = null;
```